tests/e2e/apps_test.go

```diff
@@ -142,10 +142,23 @@ var _ = Describe("Apps", func() {
 			resp, err = certClient.R().Delete("/v3/apps/" + appGUID)
 		})
 
+		It("succeeds with a job redirect", func() {
+			Expect(resp).To(SatisfyAll(
+				HaveRestyStatusCode(http.StatusAccepted),
+				HaveRestyHeaderWithValue("Location", HaveSuffix("/v3/jobs/app.delete~"+appGUID)),
+			))
+		})
+
 		It("successfully deletes the app", func() {
 			var result resource
-			Expect(resp).To(HaveRestyStatusCode(http.StatusAccepted))
-			Expect(err).NotTo(HaveOccurred())
+
```

Comment:
maybe worth ensuring the status code is `Accepted` just in case before getting the location header

Reply:
done